### ISSUE TITLE
Level Nine

### ISSUE THREAD

**Issue by vivski21** [BIGBOSSKANE/Body_Builder_Game#23]
When the player dies after getting hit by the Banshee with the arm attachment, the game is practically unplayable as the player re-spawns and cannot pass the first passage with the arm augment.

**Comment by HopscotchNinja** [comment]
1. From my experience, the first few jumps are fairly easy. I would recommend we have a few more playtesters check this, as the game is a platformer, and making it any easier would remove any risk of failure.

2. This is a respawn feature that will be implemented later (storing a prefab of the player at the checkpoint, and resetting the scene and loading them back into the checkpoint as they were). This will fix the issue, allowing the player to effectively have a fresh start.

3. This is a feature I have attempted to implement before when I get time, I will take another shot at it.

4. All non-player objects have now had their bounce heights reduced by half.

5. I've tried and cannot seem to replicate this bug. I will continue to look out for it.
